BankManagerConsoleApp/BankManagerConsoleApp/main.swift

```diff
@@ -7,4 +7,30 @@
 import Foundation
 import BankManager
 
-print(BankManager().text)
+let customerLine: LinkedList<Customer> = LinkedList()
+let customerQueue = CustomerQueue(queue: customerLine)
+let bank: Bank = Bank.init(bankClerkCount: 1, customerLine: customerQueue)
```

Comment:
여기도 취향강요인데..ㅎ
타입을 명시해줄경우에는 그냥 
```suggestion
let bank: Bank = .init(bankClerkCount: 1, customerLine: customerQueue)
```
요렇게만 쓰구 아니면 
```suggestion
let bank: Bank = Bank(bankClerkCount: 1, customerLine: customerQueue)
```
요렇게 쓰는게 뭔가 깔끔하더라구요

저는 타입추론하는 컴파일시간 아끼는 걸 좋아해서 1번을 선호합니다

Reply:
해당 방식으로 수정 완료했습니다!